webcompat/views.py

```diff
@@ -175,8 +177,14 @@ def create_issue():
             flash(msg, 'notimeout')
             return redirect(url_for('index'))
     form['ua_header'] = request.headers.get('User-Agent')
-    # Store where the report originated from
-    form['reported_with'] = request.headers.get('X-Reported-With', 'web')
+    # Currently we support either a src GET param, or X-Reported-With header
+    # to track where the report originated from.
+    # See https://github.com/webcompat/webcompat.com/issues/1254 to track
+    # supporting only the src param
+    if session['src']:
+        form['reported_with'] = session.pop('src', None)
```

Comment:
Yeah, I think we want to remove `src` from the session cookie -- it's not useful after we get its value.